### ISSUE TITLE
Line Input # not ok

### ISSUE THREAD

**Issue by KarlSevenSeven** [WaynePhillipsEA/twinbasic#542]
**Describe the bug**
Line input # reads last line in a file wrong.
Or EOF() is the problem.
Also Line Input # is quite slow compared to VB6.

**To Reproduce**
The test file is here:
https://github.com/WaynePhillipsEA/twinbasic/files/7649138/Bigfile.zip
tB file pair is here:
[LineInput.zip](https://github.com/WaynePhillipsEA/twinbasic/files/7676012/LineInput.zip)

```
Public Sub Main()
        
        Dim F As Long
        Dim m As String
          
        Dim i As Long
        
        Tick_Start 3
        F = FreeFile
        Open "Z:\BigFile.txt" For Input As #F
                
        While Not EOF(F)
            Line Input #F, m
            i = i + 1
        Wend
                
        Close #F
        
        Debug.Print "i", i
        Debug.Print "m", m
        
        Tick_Show 3, "-> Read file by line input"
           
    End Sub

```

```
tB:
i              2677049 
m             
-> Read file by line input: 8985 msec.
```

```
VB6:
i              2677048 
m             EOF
-> Read file by line input: 468 msec.
```

**Expected behavior**
Same result (and performance) as in VB6.

**Desktop**
 - OS: Windows 10
 - twinBASIC 0.13.25

**Comment by WaynePhillipsEA** [comment]
This should now be fixed in v0.13.26.

**Comment by KarlSevenSeven** [comment]
> This should now be fixed in v0.13.26.

It is; correct and even faster than VB6:
```
i              2677048 
m             EOF
-> Read file by line input: 281 msec.

```
